src/html/htmprint.cpp

```diff
@@ -559,22 +559,22 @@ wxString wxHtmlPrintout::TranslateHeader(const wxString& instr, int page)
     wxString r = instr;
     wxString num;
 
-    num.Printf(wxT("%i"), page);
-    r.Replace(wxT("@PAGENUM@"), num);
+    num.Printf("%i", page);
+    r.Replace("@PAGENUM@", num);
 
-    num.Printf(wxT("%lu"), (unsigned long)(m_PageBreaks.size() - 1));
-    r.Replace(wxT("@PAGESCNT@"), num);
+    num.Printf("%lu", (unsigned long)(m_PageBreaks.size() - 1));
```

Comment:
As long as we're modifying this, we should make it
```suggestion
    num.Printf("%zu", m_PageBreaks.size() - 1);
```